tests/test_other_formats_analysis.py

```diff
@@ -0,0 +1,61 @@
+import hashlib
```

Comment:
The name of the file though :stuck_out_tongue:

At some point, we may put all in `test_tabular_analysis.py` and dispatch these in `CsvAnalysisTest` &co classes?